packages/circuits/src/helpers/extractor.circom

```diff
@@ -2,12 +2,53 @@ pragma circom 2.1.6;
 
 include "circomlib/circuits/comparators.circom";
 include "circomlib/circuits/bitify.circom";
+include "@zk-email/circuits/utils/array.circom";
+include "@zk-email/circuits/utils/bytes.circom";
 include "../helpers/constants.circom";
-include "../utils/array.circom";
 include "../utils/pack.circom";
 
 
 
+/**
+Aadhar QR code data schema (V2)
+
+V1 Docs - https://uidai.gov.in/images/resource/User_manulal_QR_Code_15032019.pdf
+There are no official spec docs for Aadhaar V2 available publically, but the difference from V1 is:
+  - "V2" is added at the bedinning of the data, before the first delimitter.
+  - Phone and email hash is no longer present.
+  - Last 4 digits of mobile number is added (before the photo).
+
+- Delimiter is 255.
+- Before first delimiter, there are two bytes representing the version. This should be [86, 50] (V2)
+- From then on, each field is separated by the delimiter. There are total of 16 fields.
+  1 (data after first 255). Email_mobile_present_bit_indicator_value (can be 0 or 1 or 2 or 3): 
+      0: indicates no mobile/email present in secure qr code. 
+      1: indicates only email present in secure qr code. 
+      2: indicates only mobile present in secure qr code 
+      3: indicates both mobile and email present in secure qr code.
+  2. Reference ID (Last 4 digits of aadhar number and timestamp)
```

Comment:
I'm still confuse is it aadhaar? or aadhar?

Reply:
Aadhaar - this one is a type. I have fixed.